admiral/debug_functions.sqf

```diff
@@ -287,6 +287,61 @@ adm_debug_fnc_updateLineMarker = {
     DEBUG("admiral.debug",FMT_3("Updated line marker '%1' position between '%2' and '%3'.",_markerName,_posFrom,_posTo));
 };
 
+adm_debug_fnc_createDebugCounterMarkers = {
+    FUN_ARGS_2(_side,_xPos);
+
+    PVT_2(_marker,_markerType);
+    DECLARE(_debugMarkers) = [];
+    {
+        _markerType = _x;
+        if (_markerType == "total") then {
```

Comment:
This `if` should be removed in favour of iterating over an array of `[_type, _markerType]` entries.